new_platforms/vscode-extension/README.md

```diff
@@ -1,188 +1,332 @@
+  ```bash
+  # Remove CMake if already installed.
+  sudo apt remove cmake --purge
+
+  # Download the CMake tarball for Linux.
+  wget https://cmake.org/files/v3.13/cmake-3.13.1-Linux-x86_64.tar.gz
+
+  # Extract it.
+  sudo tar xf cmake-3.13.1-Linux-x86_64.tar.gz -C /usr/local
+
+  # Install CMake and its tools to the right directories.
+  sudo ln -s /usr/local/cmake-3.13.1-Linux-x86_64/bin/ccmake /usr/local/bin/ccmake
+  sudo ln -s /usr/local/cmake-3.13.1-Linux-x86_64/bin/cmake /usr/local/bin/cmake
+  sudo ln -s /usr/local/cmake-3.13.1-Linux-x86_64/bin/cpack /usr/local/bin/cpack
+  sudo ln -s /usr/local/cmake-3.13.1-Linux-x86_64/bin/ctest /usr/local/bin/ctest
+  ```
+
+* Install the required build components. On Ubuntu 18.04:
+  ```bash
+  # Flag all current directories as AMD64-only.
+  sudo sed -i 's/^deb h/deb [arch=amd64,i386] h/g' /etc/apt/sources.list
+
+  # Enable the ARM64 architecture.
+  sudo dpkg --add-architecture arm64
+
+  # Add the Intel SGX repository and key.
+  echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu bionic main' | sudo tee /etc/apt/sources.list.d/intel-sgx.list
+  wget -qO - https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | sudo apt-key add -
+
+  # Add the LLVM repository and key.
+  echo "deb [arch=amd64] http://apt.llvm.org/bionic/ llvm-toolchain-bionic-7 main" | sudo tee /etc/apt/sources.list.d/llvm-toolchain-bionic-7.list
+  wget -qO - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
+
+  # Add the Microsoft repository and key.
+  echo "deb [arch=amd64] https://packages.microsoft.com/ubuntu/18.04/prod bionic main" | sudo tee /etc/apt/sources.list.d/msprod.list
+  wget -qO - https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add -
+
+  # Add the Ubuntu ARM64 ports repository.
+  echo 'deb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports bionic main restricted universe multiverse' | sudo tee -a /etc/apt/sources.list
+  echo 'deb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports bionic-updates main restricted universe multiverse' | sudo tee -a /etc/apt/sources.list
+  echo 'deb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports bionic-backports main restricted universe multiverse' | sudo tee -a /etc/apt/sources.list
+
+  # Update the repositories.
+  sudo apt update
+
+  # Install all prerequisites.
+  sudo apt install -y apt-transport-https az-dcap-client binfmt-support      \
+    build-essential clang-7 docker.io g++-aarch64-linux-gnu                  \
+    gcc-aarch64-linux-gnu gdb gdb-multiarch libc6 libc6-dev:arm64 libfdt1    \
+    libglib2.0-0 libpcre3 libpixman-1-0 libprotobuf10 libsgx-dcap-ql         \
+    libsgx-dcap-ql-dev libsgx-enclave-common libsgx-enclave-common-dev       \
+    libssl-dev libssl-dev:arm64 libstdc++6 open-enclave python python-crypto \
+    python-pip qemu-user-static zlib1g
+  ```
+
+* If you have Intel SGX-capable hardware, install the Intel SGX DCAP driver. On
+  Ubuntu 18.04, run:
```

Comment:
Line 292 below also mentions 16.04.   Why doesn't this line?

Reply:
Added instructions for 16.04.